input/resources/au-core-server.xml

```diff
@@ -69,16 +69,16 @@
                 <p>Return the following response classes:</p>
                 <ul>
                   <li>
-                    <p>(Status 400): invalid parameter</p>
```

Comment:
These are not 'clean up' see US Core and IPA CapabilityStatements. It is a proposed shift in language away from international IGs.

We can let it pass now and I'll raise a JIRA to shift it back. It can be discussed in TDG.

Reply:
noted